src/nodejs/config/config.js

```diff
@@ -50,3 +52,16 @@ exports.get_smtp = () => {
     return smtp
   }
 }
+
+// Load a list of nodes from configuration file
+// Net can be any name  you provide as they sub key under "nodes" in the config
+
+exports.get_nodes = (net) => {
+  var nodes = cfg.nodes
+  if(nodes && nodes.path !== null) {
+    var nodesCfg = require(nodes.path)
+    return nodesCfg.nodes[net]
+  } else {
+    return nodes.nodes[net]
```

Comment:
You're not sure here whether nodes exists, right?

Reply:
Nope, will fix.